tests/views/index.jade

```diff
@@ -1,12 +1,14 @@
 extends layout
 
 append scripts
+    script(src='../node_modules/babel-polyfill/dist/polyfill.js')

```

Comment:
I just added `Array.prototype.includes` in this PR--more semantic and clean than `indexOf`.

So do you want the polyfill for the sake of IE/Edge (in case it may work there and somebody actually wants to run tests in it)?